tools/fixFormat.js

```diff
@@ -67,6 +67,21 @@ async.each(packages, function(item, callback) {
     pkg.licenses = pkg.license;
     delete pkg.license;
   }
+  if (pkg.npmFileMap) {
+    for (var i in pkg.npmFileMap) {
+      var npmbasePath = pkg.npmFileMap[i].basePath;
+      if (npmbasePath && npmbasePath.length != 0 && ((typeof npmbasePath) == "string")) {
+          npmbasePath = (npmbasePath).replace(/^\/|\/$/g , "");
```

Comment:
@PeterDaveHello @cdnjs/intern2 I have fixed using `/^\/+|\/+$/g`. Please help me review it again, thank you.